### DISCUSSION TITLE
V2.9 Installation

### DISCUSSION THREAD

**Discussion by dpsrikanth** [egovernments/Digit-Core#269] [DIGIT Platform]
What version of DIGIT are you using?
V2.9

Are you attempting quickstart (single node deploy) or full installation of Digit?
Full Installation

Is this installed using a laptop or AWS/GCP/Azure or State Data Centre?
AWS

Please provide link to the documentation used while installing.
https://core.digit.org/guides/installation-guide/quick-setup/digit-installation-on-aws

Please provide specific step number where error was encountered.
Step 12

What is the OS? Provide version as well. For eg. Ubuntu 20.04
Amazon Linux 2 

Number of cores/vCPUs and RAM available?
12vCPUs 96GB 

Free RAM currently available.(How to find free memory - )
90GB

Description of the error along with a screenshot. Please paste the exact error encountered on the screen.
Kubernetes logs (if applicable) or other relevant logs.

Digit deployment was successful but "kubectl get pods -A" is not showing digit related pods. It is only showing kube-system related pods.

**Comment by nikhilmulinti** [comment]
@dpsrikanth Please let us know if you are facing any installation related issues
